src/serializer/serializer.js

```diff
@@ -1748,6 +1833,6 @@ export class Serializer {
     let serialized = this.serialize(filename, code, sourceMaps);
     invariant(!this.logger.hasErrors());
     if (this.options.logStatistics) this.statistics.log();
+    serialized.generated.statistics = this.statistics;
```

Comment:
That seems fine for exposing statistics, then you can just access it from the `serialized` object in the `test-runner`.